src/target/riscv/riscv.c

```diff
@@ -619,8 +619,17 @@ static int find_first_trigger_by_id(struct target *target, int unique_id)
 	return -1;
 }
 
+static unsigned int count_trailing_ones(riscv_reg_t reg)
+{
+	for (int i = 0; i < 64; i++) {
```

Comment:
Very small detail:

This helper function assumes that `riscv_reg_t` is an alias to `uint64_t`. 

Please, consider putting an assertion there:

```suggestion
	assert(sizeof(riscv_reg_t) * 8 == 64);

	for (int i = 0; i < 64; i++) {
```

Reply:
addressed